### PR TITLE
COMMERCE-10173

### PR DESCRIPTION
**Pull request by brianikim**
From @victorcdp ,
> Related Issue: [COMMERCE-10173](https://issues.liferay.com/browse/COMMERCE-10173)
> 
> Problem was that there was no validation of existence in the inputs. Proposed fix is to add a required validator so that the product can't be saved with the specified inputs empty. Also there was no validation at the backend, so this solution includes that as well.

@andrea-ale-sbarra

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CPDefinitionInventoryLocalServiceImpl.java [hunk 1]
```diff
@@ -209,6 +216,19 @@ public CPDefinitionInventory updateCPDefinitionInventory(
...
+	private void _validateOrderQuantity(
+			int minOrderQuantity, int maxOrderQuantity,
+			int multipleOrderQuantity)
+		throws CommerceOrderQuantityException {
+
+		if ((minOrderQuantity < 1) || (maxOrderQuantity < 1) ||
+			(multipleOrderQuantity < 1)) {
+
+			throw new CommerceOrderQuantityException(
+				"Order Quantity must be greater than  or equal to 1");
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CPDefinitionInventoryLocalServiceImpl.java:228]: I don't think you are giving the right message doing like this, you should have an if case for each quantity
```suggestion
		if ((minOrderQuantity < 1){

			throw new CommerceOrderQuantityException(
				"Minimum Order Quantity must be greater than  or equal to 1");
				}
		if( maxOrderQuantity < 1) {
					throw new CommerceOrderQuantityException(
				"Maximum Order Quantity must be greater than  or equal to 1");
				}
				if( multipleOrderQuantity < 1) {
					throw new CommerceOrderQuantityException(
				"Multiple Order Quantity must be greater than  or equal to 1");
				}
		

```

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CPDefinitionInventoryLocalServiceImpl.java (hunk 1 above)
Comment (brianikim) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CPDefinitionInventoryLocalServiceImpl.java:228]: @victorcdp , see requested change.

Comment (brianikim) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CPDefinitionInventoryLocalServiceImpl.java]: @andrea-ale-sbarra , required changes have been made.

**Comment by brianikim** [comment]
@andrea-ale-sbarra , conflict merge addressed.

**Comment by andrea-ale-sbarra** [comment]
@brianikim see here https://github.com/brianikim/liferay-portal/pull/69

**Pull request closed without merging**